src/types/service/MatchHistory/dataMappers.ts

```diff
@@ -85,6 +85,12 @@ export function sheetRowToMatch(cell: SheetRow, id: string): Match {
     };
 }
 
+/**
+ * Gets the winrate of a player from a collection of matches
+ * @param matches Automatically filtered to valid matches (by player count)
```

Comment:
this is not true :) When we call "GetPlayerWinRate", the function cannot guarantee that the matches are valid. In fact, you specifically do a check inside this method that does the filtering. 

So you can replace this comment something like:
"@param matches The collection of matches to determine the win rate against."

And you could update the function description, but I think it is unnecessary: 
" * Gets the winrate of a player from a collection of matches. This collection of matches can be a mix of valid and invalid matches as determined by the number of players in that match.

Reply:
fixed the @param description to your suggestion.

however...

_Gets the winrate of a player from a collection of matches. This collection of matches can be a mix of valid and invalid matches as determined by the number of players in that match._

i don't think this is accurate either. the collection can be a mix but that doesn't determine anything. the output is simply filtered from the inputs automatically. maybe:

_Gets the winrate of a player from a collection of matches. This collection of matches can be a mix of valid and invalid matches. Invalid matches get filtered out automatically._

or we could move the "can be a mix of valid and invalid matches" to the @param description because that describes the input really well.

Reply:
Your response here is good, but I would recommend rewording the " Invalid matches get filtered out automatically." part. Notably, nothing really happens "automatically" ,it's the function doing things. You could add that sentence to the end of the first part, resulting in : 

Gets the winrate of a player from a collection of matches without considering invalid matches.

And then move this into the @param description: 
 The collection of matches can be a mix of valid and invalid matches. 
 as you said in your reply :D

Reply:
fixed